includes/Plugin.php

```diff
@@ -154,7 +154,12 @@ function() use ( $options, $activation_flag ) {
 				$user_options = new Core\Storage\User_Options( $this->context, get_current_user_id() );
 				$assets       = new Core\Assets\Assets( $this->context );
 
-				$authentication = new Core\Authentication\Authentication( $this->context, $options, $user_options, $transients );
+				$survey_queue = new Core\User_Surveys\Survey_Queue( $user_options );
+				$survey_queue->register();
+
+				$user_input = new Core\User_Input\User_Input( $this->context, $options, $user_options, $survey_queue );
```

Comment:
One thing that we forgot to do is to add `$user_input->register()` here instead of keeping it in the `Authentication` classs.

Reply:
Just updated this too, thanks so much!